EIPS/eip-1775.md

```diff
@@ -0,0 +1,571 @@
+
+
+
+## Test Cases
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+[TBD]
+
+Provide some examples of accounts derived from a given mnemonic, persona, application and application's custom subpath.
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+[WIP]
+[See here for an early implementation of the HD methods](https://github.com/MetaMask/KeyringController/blob/plugin-system/index.js)
+
+## Example use cases:
+
+* signing transactions without broadcasting them:
+https://github.com/MetaMask/metamask-extension/issues/3475
+
+* token contract:
+https://github.com/ethereum/EIPs/issues/85
+
+* default account for dapps
+https://ethereum-magicians.org/t/default-accounts-for-dapps/904
+
+* non wallet/crypto accounts
+[EIP 1581: Non-wallet usage of keys derived from BIP-32 trees](https://eips.ethereum.org/EIPS/eip-1581)
+
+* state channel application
+
+* privacy solution
+
+* non custodian cross cryptocurrency exchange...
+
+## Acknowledgements
+MetaMask team, Christian Lundkvist, Counterfactual team, Liam Horne, Erik Bryn, Richard Moore, Jeff Coleman.
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+## References:
+
+### HD and mnemonics
+#### BIPS:
+[BIP 32 specs, Hierarchical Deterministic Wallets:](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
+
+[BIP 39 specs, Mnemonic code for generating deterministic keys:](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
+
+[BIP43: Purpose Field for Deterministic Wallets](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki)
+
+[BIP44: Multi-Account Hierarchy for Deterministic Wallets](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Address_gap_limit)
+
+[SLIP44: Registered coin types for BIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
+
+[Is there a comprehensive list of registered BIP43 purposes?](https://bitcoin.stackexchange.com/questions/60470/is-there-a-comprehensive-list-of-registered-bip43-purposes)
+
+#### Derivation path for eth:
+[ERC84](https://github.com/ethereum/EIPs/issues/84)
+
+[ERC85](https://github.com/ethereum/EIPs/issues/85)
+
+
+Draft EIP600 Ethereum purpose allocation for Deterministic Wallets:
```

Comment:
These EIP pull requests #600 and #601 are being updated as I type this, so when updating the Draft pull request you might be able to link directly to these on the EIP website?

Reply:
Ok, for now I'm changing the commits links for the PR links, since it's not up on the EIP website yet.

Reply:
https://eips.ethereum.org/EIPS/eip-601 is available
https://eips.ethereum.org/EIPS/eip-600 was merged, but accidentally not into the correct folder. That is fixed.

Reply:
ok done.